index.html

```diff
@@ -25,54 +31,44 @@
+                                <option value='link.html'>&lt;link></option>
+                                <option value='meta.html'>&lt;meta&gt;</option>
+                                <option value='style.html'>&lt;style></option>
+                                <option value='title.html'>&lt;title></option>
+                              </select>
+                            </div>
                         </li>
                     </ul>
                 </div>
             </div>
         </nav>
     </div>
-    <h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
-    <div class='droplist'>
-        <label for='tags'></label>
-        <select name='tags' id='tags'>
-            <option selected value="empty.html">Select a tag/Selecione una etiqueta</option>
-            <option value='a.html'>&lt;a></option>
-            <option value='base.html'>&lt;base></option>
-            <option value='body.html'>&lt;body></option>
-            <option value='br.html'>&lt;br></option>
-            <option value='footer.html'>&lt;footer></option>
-            <option value='head.html'>&lt;head&gt;</option>
-            <option value='header.html'>&lt;header></option>
-            <option value='html.html'>&lt;html></option>
-            <option value='link.html'>&lt;link></option>
-            <option value='meta.html'>&lt;meta&gt;</option>
-            <option value='style.html'>&lt;style></option>
-            <option value='title.html'>&lt;title></option>
-        </select>
-    </div>
-    <main>
-        <div class="main-inside inside" id="main">
-        </div>
-    </main>
-    <footer class="site-footer">
-        Project by Team Open Brace, From <a href="http://ywebca.org/" target="_blank">YWebCA</a> Summer 2015.
-        <br> Download the template HTML file for making your own card <a href="elements/template.html" download>here</a>.
-        <br> Check out our <a href="https://github.com/kevinbrianfahy/LMHT" target="_blank">GitHub repository</a> to send a pull request for your card!
-    </footer>
-    <script src='js/jquery-1.11.3.min.js'></script>
-    <script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
-    <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
-    <script src='js/scripts.js'></script>
+<main>
+  <div class="main-inside inside" id="main">
+  </div>
+</main>
+  <footer class="site-footer">
+    Project by Team Open Brace, From <a href="http://ywebca.org/" target="_blank">YWebCA</a> Summer 2015.<br>
+    Download the template HTML file for making your own card <a href="elements/template.html" download>here</a>.<br>
+    Check out our <a href="https://github.com/kevinbrianfahy/LMHT" target="_blank">GitHub repository</a> to send a pull request for your card!
```

Comment:
I think it would be more semantic to use `<p>`s here rather than `<br>`s. You can get rid of the margin with CSS if you don't like it.